recipe/meta.yaml

```diff
@@ -20,20 +20,18 @@ requirements:
     - python
     - binaryen
     - nodejs
-    # See https://github.com/emscripten-core/emscripten/blob/2.0.8/tools/shared.py#L40
-    - lld        12.*
-    - clang      12.*
-    - clangxx    12.*
-    - llvm-tools 12.*
+    - lld
+    - clang
+    - clangxx
+    - llvm-tools
   run:
     - python
     - {{ pin_compatible("binaryen", max_pin="x") }}
     - nodejs
-    # See https://github.com/emscripten-core/emscripten/blob/2.0.8/tools/shared.py#L40
-    - lld        12.*
-    - clang      12.*
-    - clangxx    12.*
-    - llvm-tools 12.*
+    - lld
```

Comment:
Looks like expected LLVM is 15.0 and minimum node version is 4.1: https://github.com/emscripten-core/emscripten/blob/a23058a54d506a104e42dfe348e0d53233c409c6/tools/shared.py#L48-L49

Reply:
Yes, this seems to only trigger a warning though 
```
emcc: warning: LLVM version for clang executable "/home/martin/micromamba/envs/emscripten-feedstock/conda-bld/emscripten_1651659645629/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/bin/clang" appears incorrect (seeing "13.0", expected "15.0") [-Wversion-check]
```

It seems we need to find the right versions combination between binaryen, llvm, clang and emscripten